### PR TITLE
slack signup page with GDPR incorporated

### PR DESCRIPTION
**Pull request by VineeshJain**
Added the code to incorporate the GDPR compliancy (Country and Consent checkbox) for slack signup page.

### CODE REVIEW

File: record_ac/handler.py [hunk 1]
```diff
@@ -65,17 +73,17 @@ def add_ac_contact(config, email, first_name='', last_name='', tags=''):
...
 
     if resp.status_code == 200 and result['result_code'] != 0:
-        print "SUCCESS"
+        print ("SUCCESS")
     else:
         failure_reason = (
             'Failed to create AC contact for {}: {} (status code: {})'.format(
                 email, resp.text, resp.status_code))
-        print failure_reason
+        print (failure_reason)
```
Comment (LindsayHill) [inline record_ac/handler.py]: Minor nit-pick - looks like whitespace at the end of the line

File: web/index.html [hunk 2]
```diff
@@ -43,9 +43,268 @@ <h2 class="f5 tc f3-l fw1 mb4 near-white lh-title">Welcome to StackStorm! </h2>
...
+					</select>
+                </div>
+                <div class="mt1">
+					<input type="checkbox" name="consent" value="true" id="consent-checkbox">
+					<label for="consent-checkbox">
+      					<span><font color="white">Please check the box if you are happy for StackStorm to keep you updated about our latest products, services and promotions via email.</span>
+   					</label>
                 </div>
               </fieldset>
+              <a href="https://stackstorm.com/privacy-policy//">Privacy Policy</a>
```
Comment (LindsayHill) [inline web/index.html]: Minor nitpick - you've got an extra `/` at the end of the URL. Should be single `/` at the end.

**Review by LindsayHill — APPROVED** [review]
Couple of minor things, but otherwise looks good

File: record_ac/handler.py (hunk 1 above)
Comment (VineeshJain) [reply record_ac/handler.py]: Fixed

File: web/index.html (hunk 2 above)
Comment (VineeshJain) [reply web/index.html]: Thanks. Fixed

File: tests/record_ac_test.py [hunk 3]
```diff
@@ -7,16 +7,18 @@
...
+    'country': 'United States',
+    'consent': 'true'
 }
 
 context = {}
 
 env = {
     'URL': 'https://stackstorm.api-usa.com',
-    'API_KEY': '9042624c1641a25936c93fa2fae9986f8b1af950de876485c785dbdb75f0b9af01fc08f4'
+    'API_KEY': 'bfa6d1f199fbf74508509aea59b6df6b0e84833f605af9c9b92778e61c91e10f5ed5133c'
```
Comment (dzimine) [inline tests/record_ac_test.py:21]: I think we must remove this from here and use `os.env.get('API_KEY')` instead.

**Review by dzimine — APPROVED** [review]

**Pull request merged**